public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-admin.php

```diff
@@ -706,6 +707,56 @@ public function mark_date_added_to_planning_schedule( $wordcamp ) {
 			update_post_meta( $wordcamp->ID, '_timestamp_added_to_planning_schedule', time() );
 		}
 
+		/**
+		 * Send notification to slack when a WordCamp is scheduled or declined. Runs whenever status of an applications changes
+		 *
+		 * @param string  $new_status
+		 * @param string  $old_status
+		 * @param WP_Post $wordcamp
+		 *
+		 * @return null|bool
+		 */
+		public function notify_application_status_in_slack( $new_status, $old_status, WP_Post $wordcamp ) {
+
+			$notification_enabled = apply_filters( 'wordcamp_application_notification_enabled', true );
+
+			if ( ! $notification_enabled ) {
+				return null;
+			}
+
+			if ( 'wcpt-scheduled' === $new_status ) {
+				return $this->notify_new_wordcamp_in_slack( $wordcamp );
+			} elseif ( 'wcpt-rejected' === $new_status ) {
+				$location = get_post_meta( $wordcamp->ID, 'Location', true );
+				return $this->schedule_decline_notification( $wordcamp, 'WordCamp', $location );
+			}
+		}
+
+		/**
+		 * Send notification when a new WordCamp comes in scheduled status.
+		 *
+		 * @param WP_Post $wordcamp
+		 *
+		 * @return null|bool
+		 */
+		public static function notify_new_wordcamp_in_slack( $wordcamp ) {
+			// Not translating any string because they will be sent to slack.
+			$city             = get_post_meta( $wordcamp->ID, 'Location', true );
+			$start_date       = get_post_meta( $wordcamp->ID, 'Start Date (YYYY-mm-dd)', true );
+			$wordcamp_url     = get_post_meta( $wordcamp->ID, 'URL', true );
+			$title            = 'New WordCamp scheduled!!!';
+
+			$message = sprintf(
+				"WordCamp $city has been scheduled for %s. :tada: :community: :wordpress:\n\n%s",
+				date( 'd-M-Y', $start_date ),
```

Comment:
I think the `'d-M-Y'` date format will be confusing to some people (at least in the US). It might be better to go with something more longform like `January 9th, 2019`.

Also, should we include the end date here too for multiday camps?

Reply:
Maybe something like `WordCamp $city has been scheduled for a start date of %s.`

Another thought, what if we linked `WordCamp $city` to the event site?

Reply:
I think we should do both. show a URL as well as link it.